packages/server/test/unit/browsers/chrome_spec.coffee

```diff
@@ -70,16 +70,17 @@ describe "lib/browsers/chrome", ->
 
     it "does not load extension in headless mode", ->
       plugins.has.returns(false)
+      chrome._writeExtension.restore()
 
       pathToTheme = extension.getPathToTheme()
 
-      chrome.open("chrome", "http://", { isHeadless: true, isHeaded: false }, @automation)
+      chrome.open({ isHeadless: true, isHeaded: false }, "http://", {}, @automation)
```

Comment:
I feel here it would be helpful to extract arguments into named objects like maybe
```
browserInfo = {
  isHeadless: true,
  isHeaded: false
}
urlToLoad = "http://"
options = {}
chrome.open(browserInfo, urlToLoad, options, @automation)
```